sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/HiveMetastoreLazyInitializationSuite.scala

```diff
@@ -32,6 +32,8 @@ class HiveMetastoreLazyInitializationSuite extends SparkFunSuite {
       .master("local[2]")
       .enableHiveSupport()
       .config("spark.hadoop.hive.metastore.uris", "thrift://127.0.0.1:11111")
+      .config("spark.ui.enabled", "false")
+      .config("hive.metastore.connect.retries", "1")
```

Comment:
Could you clarify why do you need this, please. Won't it lead to test flakiness?

Reply:
+1 for @MaxGekk 's comment.

Reply:
Hi @MaxGekk @dongjoon-hyun, as `thrift://127.0.0.1:11111` is an unreachable HMS endpoint, it's better to fail at once w/o unnecessary retries to reduce the test time. It will not increase the flakiness.

Reply:
Another thing is that this SparkSession instance is not for sharing and will be closed in the final block.

Reply:
I think the purpose of this configuration is to allow for fast failure in order to save some test time, right? Even without this modification, the test can still succeed, correct?

Reply:
Yes, the purpose here is the same as `ui.enabled` for lightweight testing. There is no increase in risk of flakiness here. It's just a piggybacking change. I did this regarding to low possibility of future changes against this file. It's also OK to remove them and leave it as-is if you insist.

Reply:
Fail-fast is not a bug fix. To be clear, please proceed that first before this PR, @yaooqinn .

Reply:
Thanks @dongjoon-hyun, removed

Reply:
Since this is restored, the PR is only one line change, I will merge it now